Classes/Command/SortingCommand.php

```diff
@@ -31,6 +31,12 @@ protected function configure()
     {
         $this->addArgument('dryrun', InputArgument::OPTIONAL, 'do not execute queries', true);
         $this->addOption('apply', null, InputOption::VALUE_NONE, 'apply migration');
+        $this->addOption(
+            'enable-logging',
```

Comment:
I feel like this option should be called "--debug"

Reply:
hm, but it debugs nothings, but it logs?, show i feel it should called enable-logging?